backend/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@octokit/rest": "^21.0.2",
     "axios": "^1.6.3",
+    "backend": "file:",
     "bcrypt": "^5.1.1",
```

Comment:
@Arpin007 why did you added the backend dependency. It is not used anywhere.
Can you remove it if not required.